jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-kops-aws.yaml

```diff
@@ -0,0 +1,67 @@
+- job-template:
+    name: 'kubernetes-e2e-kops-{suffix}'
+    node: '{jenkins_node}'
+    description: '{description} Test owner: {test-owner}.'
+    jenkins_node: 'e2e'
+    disabled: '{obj:disable_job}'
+    properties:
+        - build-discarder:
+            days-to-keep: 7
+    triggers:
+        - reverse:
+            jobs: '{trigger-job}'
```

Comment:
Consider removing as many of these as appropriate (we only trigger one way, we only define one version of emails)